### DISCUSSION TITLE
Show username in compact post layout

### DISCUSSION THREAD

**Discussion by Ep0chalypse** [Docile-Alligator/Infinity-For-Reddit#958] [Ideas]
I use the compact layout for posts and it would be nice to see the username of the poster when scrolling. It would probably fit between the subreddit name and post time. If not, placing it underneath the subreddit name would work as well.

**Comment by mmcnutt** [comment]
I would also like to see this feature added. It could probably fit between the tags and the up/down buttons, without adding much size to the compact view

**Comment by snvoid** [comment]
Also chiming in that this would be a valuable feature to have. I'm coming from RIF and that shows the username in compact mode and I find it useful to be able to see at a glance who the person making the post is.
